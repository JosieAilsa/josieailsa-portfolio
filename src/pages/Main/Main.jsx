import Home from "../Home/Home";
import Layout from "../../components/Layout/Layout"
import Projects from "../Projects/Projects";
import Skills from "../Skills/Skills"

const Main = () => {
    return (
        <Layout>
            <Home/>
            <Projects/>
        </Layout>
    );
}
 
export default Main;