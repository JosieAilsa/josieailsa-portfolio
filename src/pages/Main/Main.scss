@use "../../assets/sass/abstracts/variables" as *;

#landing-text{
    font-family: $font-accent;
    font-style: italic;
    font-weight: 800;
    font-size: 5rem;
}