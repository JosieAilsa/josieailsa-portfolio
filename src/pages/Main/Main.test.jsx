import { customRender } from "../../utils/testutils";
import Main from "./Main";

test("Main page should render", () => {
  const { container } = customRender(<Main />);
  expect(container).toMatchSnapshot();
});
