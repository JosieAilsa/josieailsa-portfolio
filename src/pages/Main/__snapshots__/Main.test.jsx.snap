// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main page should render 1`] = `
<div>
  <header>
    <nav
      class="navbar"
    >
      <img
        alt="logo"
        class="navbar__logo"
        src="jnlogo-02.svg"
      />
      <ul
        class="navbar__links"
      >
        <li
          class="navbar__link"
        >
          projects
        </li>
        <li
          class="navbar__link"
        >
          skills
        </li>
        <li
          class="navbar__link"
        >
          contact
        </li>
      </ul>
      <img
        alt="nav-menu"
        class="navbar__hamburger"
        src="hamburger.svg"
      />
    </nav>
  </header>
  <div
    class="home page"
  >
    <section
      class="hero"
    >
      <section
        class="hero__text"
      >
        <h1
          class="hero__header"
        >
          Hi, I'm Josie.
        </h1>
        <h3
          class="hero__intro"
        >
          I'm Bristol-based UI developer and software coach.
        </h3>
        <button
          class="hero__button hero__button--active"
          role="button"
        >
          See my projects
        </button>
      </section>
      <img
        alt="hero image"
        class="hero__image"
        src="landing-img.svg"
      />
    </section>
  </div>
  <div
    class="projects page"
  >
    <h1
      class="projects__heading"
    >
      Projects
    </h1>
    <section
      class="carousel"
    >
      <div
        class="carousel__buttons"
      >
        <button
          class="carousel__button"
          role="button"
        >
          <img
            alt=""
            src="arrow.svg"
          />
        </button>
        <button
          class="carousel__button"
          role="button"
        >
          <img
            alt=""
            src="arrow.svg"
          />
        </button>
      </div>
      <div
        class="carousel__copy"
      >
        <h3
          class="carousel__header"
        >
          Project Name
        </h3>
        <p
          class="carousel__text"
        >
          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quia possimus minima maiores eos iusto quam ut exercitationem sit assumenda, corporis cum incidunt aliquam facere placeat, qui aliquid quod esse deleniti.
        </p>
      </div>
    </section>
  </div>
</div>
`;
